src/api/Controllers/CoursesController.cs

```diff
@@ -97,6 +97,11 @@ public IActionResult Index([FromBody]IList<Course> courses)
             //
             IActionResult result = BadRequest();
 
+            if (courses.Count() < 1)
+            {
+                throw new Exception($"Cowardly refusing to import {courses.Count()} courses to avoid wiping the database (again)");
```

Comment:
Do we actually need to throw exception? Can we not just ignore empties? And log the fact that emptiness is not wanted?

Reply:
I think an exception is a reasonable choice for this situation. Clearly something’s gone really wrong if we’re publishing an empty set of courses - an exception should lead to the process failing, an alert happening and the team investigating the failure. Simply ignoring this case is a silent failure that will likely mask the real underlying problem.